playbooks/openshift/README.md

```diff
@@ -76,15 +78,22 @@ Decide a name for your cluster, create a new directory and copy the example conf
     $ cp ~/git/pouta-ansible-cluster/playbooks/openshift/example_cluster_vars.yaml cluster_vars.yaml
 
 Change at least the following config entries:
+
     cluster_name: "YOUR_CLUSTER_NAME" 
     ssh_key: "bastion-key"
-
+    openshift_public_hostname: "your.master.hostname.here"
+    openshift_public_ip: "your.master.ip.here"
```

Comment:
Good points, project_external_ips will have to be populated. Certificates are optional (self signed ones will be generated if none are given) and bastion_secgroup according to the instructions is called 'bastion'.